polaris-react/src/components/Filters/Filters.tsx

```diff
@@ -91,6 +91,8 @@ export interface FiltersProps {
   disableFilters?: boolean;
   /** Whether the text field should be borderless. Should be true when used as part of the IndexFilters component. */
   borderlessQueryField?: boolean;
+  /** Prefix SearchMinor icon on query field. */
+  showQueryIcon?: boolean;
```

Comment:
yeah I think making the experiences consistent with only the search icon makes sense for now, but we can extend in the future if needed!

Reply:
@davidlng just shared [this issue](https://github.com/Shopify/web/issues/108834) with me, I'm think I'll just remove this prop and build it in for all tables again since it was just removed for IndexFilters anyways and if they want to add it back in we can add it back in everywhere